packages/smooth_app/lib/pages/product/product_image_swipeable_view.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:flutter/material.dart';
+import 'package:smooth_app/data_models/product_image_data.dart';
+import 'package:smooth_app/generic_lib/design_constants.dart';
+import 'package:smooth_app/generic_lib/widgets/smooth_back_button.dart';
+import 'package:smooth_app/pages/product/product_image_viewer.dart';
+
+///Widget to display swipeable product images,
+///Opens product image with [initialProductImageDataIndex] from list of images Typecasted from [selectedImages]
+///
+///Field [selectedImages],[initialProductImageDataIndex],[barcode] cannot be null
+class ProductImageSwipeableView extends StatefulWidget {
+  const ProductImageSwipeableView({
+    Key? key,
```

Comment:
You can write `super.key` instead.
Personally I don't believe in `key`, but that's another story.

Reply:
Okay